app/src/main/java/mozilla/lockbox/presenter/FxALoginPresenter.kt

```diff
@@ -77,6 +80,10 @@ class FxALoginPresenter(
     private fun triggerOnboarding() {
         if (fingerprintStore.isFingerprintAuthAvailable) {
             dispatcher.dispatch(RouteAction.Onboarding.FingerprintAuth)
+        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            if (SettingStore.shared.autofillAvailable) {
+                dispatcher.dispatch(RouteAction.Onboarding.Autofill)
+            }
```

Comment:
What happens in Android versions at or above `O`, but where `autofillAvailable` is `false`?

Reply:
I believe that's what it should do. If autofill isn't available then we shouldn't tell them to use autofill 🤔